styles/globals.css

```diff
@@ -3,6 +3,16 @@
   src: local("AlmondCream"), url(../public/AlmondCream.woff) format("woff");
 }
 
+:root {
+  --mainbackground-color: #50c4b7;
```

Comment:
Maybe name this `--main-background-color` or `--primary-color`, because there is a secondary color in line 8?